client-data/js/board.js

```diff
@@ -501,16 +501,23 @@ Tools.getColor = (function color() {
 
 Tools.colorPresets.forEach(Tools.HTML.addColorButton.bind(Tools.HTML));
 
-Tools.getSize = (function size() {
+Tools.setSize = (function size() {
 	var chooser = document.getElementById("chooseSize");
+
 	function update() {
 		chooser.value = Math.max(1, Math.min(50, chooser.value | 0));
 	}
 	update();
+
 	chooser.onchange = chooser.oninput = update;
-	return function () { return chooser.value; };
+	return function (value) {
+		if (value) { chooser.value=value; update(); }
```

Comment:
```suggestion
		if (value!=null) { chooser.value=value; update(); }
```

Reply:
Why would you only check `null`? We should also cover `undefined`. And since `0` is not a valid size anyway this check should work.

Reply:
`!=null` covers undefined. This makes it explicit that we are checking for the case where the function was called without arguments. And this changes the behavior of `setSize(0)` to use the minimum size.

Reply:
I prefer not having any type coercions (except for checking if a value is true-ish which tends to be intentional). If we check for `null` and `undefined` we should make it explicit.